Makefile

```diff
@@ -9,7 +32,7 @@ CHARTDIR ?= kubernetes
 CHART_METADATA_IMAGE ?= artifactory.algol60.net/csm-docker/stable/chart-metadata
 YQ_IMAGE ?= artifactory.algol60.net/docker.io/mikefarah/yq:4
 HELM_IMAGE ?= artifactory.algol60.net/docker.io/alpine/helm:3.7.1
-HELM_UNITTEST_IMAGE ?= artifactory.algol60.net/docker.io/quintush/helm-unittest
+HELM_UNITTEST_IMAGE ?= artifactory.algol60.net/docker.io/quintush/helm-unittest:latest
```

Comment:
We usually try to avoid the `latest` tag, but since it didn't have a tag, it would use `latest` anyway, this just makes it more explicit.  Should we instead pin it so we don't get any future surprises?  It looks like the latest one was built over a year ago, and is `3.11.2-0.3.0`.  See https://hub.docker.com/layers/quintush/helm-unittest/3.11.2-0.3.0/images/sha256-6e1e699fc6001103f6419b389c4bca9b61076b11e89246443b33ee3127ad7678?context=explore

Reply:
Yeah, it's known issue with `helm-unittest`, they don't have backward compatible rolling tag, like `v0.2` and `v0.3`: https://hub.docker.com/r/quintush/helm-unittest/tags. Given that this is build time dependency only, it was considered a lower criticality - in case if incompatible change happens in `latest` (that is exactly what happened) we just need to fix a build. Using fixed manifest is no better in this sense, original image can be rebuilt and manifest will go away, breaking our build.

Reply:
Okay, sounds good.  Thanks for the explanation.

Reply:
You are right, `latest` feels like an eyesore ... I noticed that here we are using a true docker.io cache, not a locally rebuilt signed image:

    artifactory.algol60.net/docker.io/quintush/helm-unittest:latest

vs

    artifactory.algol60.net/csm-docker/stable/docker.io/quintush/helm-unittest:latest

So we can actually fix upstream tags for `helm-unittests` (and `helm`, to make it compatible), to avoid at least breaking due to incompatibility in future.